src/index.html

```diff
@@ -11,12 +11,110 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a
+          href="#"
+          class="logo_link"
+        >
+          <img
+            src="./images/logo.png"
+            alt="logo"
+            class="logo_moyo"
+          />
+        </a>
+      </div>
```

Comment:
Please add empty lines between multiline sibling elements - here, in `<ul>` and in `<head>`.